sources/updater/updog/src/main.rs

```diff
@@ -424,6 +444,31 @@ fn initiate_reboot() -> Result<()> {
     Ok(())
 }
 
+/// Returns the target version. `force_version` takes priority. If that's not specified, take it from the config.
+fn get_forced_version(config: &Config, force_version: &Option<Version>) -> Result<Option<Version>> {
+    // If a version is specified on the command line, return it as the target version
+    if let Some(version) = force_version {
+        Ok(Some(version.to_owned()))
+    // If no version is specified on the command line, get the target version from the config
+    } else {
+        // If the version is locked to "latest", we return None for the target version to indicate that
+        if config.version_lock == "latest" {
+            Ok(None)
+        } else {
```

Comment:
```suggestion
    if let Some(version) = force_version {
        return Ok(Some(version.to_owned()));
    }
    // If no version is specified on the command line, get the target version from the config
    // If the version is locked to "latest", we return None for the target version to indicate that
    } else if config.version_lock == "latest" {
        return Ok(None);
    }
```
I don't think we need the nesting/indents.